crates/bevy_sprite/src/texture_atlas_builder.rs

```diff
@@ -174,7 +174,6 @@ impl<'a> TextureAtlasBuilder<'a> {
     /// # use bevy_sprite::prelude::*;
     /// # use bevy_ecs::prelude::*;
     /// # use bevy_asset::*;
-    /// # use bevy_render::prelude::*;
```

Comment:
This should be added back; the `Image` type from `bevy_render` is used in this example for the texture assets resource.

Reply:
Good catch, changing `bevy_sprite` was an accident and explains how I missed this in testing 😅

This is now fixed in ae8320d. I've since run `cargo check --all` and it didn't find any other issues.